examples/standalone/README.md

```diff
@@ -260,6 +259,55 @@ iam.googleapis.com \
 --project ${PROJECT_ID}
 ```
 
+### Outputs by personas
+
+These outputs can be interesting for all user:
+
+- Blueprint type
+- Data ingestion bucket name
+- Data ingestion dataflow bucket name
+- Confidential data project id
+- Data governance project id
+- Data ingestion project id
+- Non-confidential data project id
+- Service account: dataflow controller service account
+
+These outputs can be interesting for data analyst group:
```

Comment:
the data analyst if likely interested in the output (i.e the BQ datasets) and data lineage things too rathan than only the input.  The input is usually plumbed by the data engineer.